odl/space/ntuples.py

```diff
@@ -31,13 +31,12 @@
 from future.utils import native
 
 # External module imports
-# pylint: disable=no-name-in-module
 import ctypes
 from numbers import Integral
 from functools import partial
 from math import sqrt
 import numpy as np
-import scipy as sp
+import scipy.linalg as linalg
```

Comment:
Avoid importing `a.b.c`, it harms mock and is somewhat irritating to deal with in code since it is not obvious what `linalg` is intended. Simply use `sp.linalg` instead.